test/unit/processing/siti-agri/handle-known-defects/cap-bps-penalties.test.js

```diff
@@ -0,0 +1,43 @@
+const { capBPSPenalties } = require('../../../../../app/processing/siti-agri/handle-known-defects/cap-bps-penalties')
+const { bps, sfi } = require('../../../../../app/schemes')
+const { P02, P04 } = require('../../../../../app/constants/line-descriptions')
+
+jest.mock('../../../../../app/processing/siti-agri/handle-known-defects/cap-bps-penalties/recalculate-bps-penalties')
+const { recalculateBPSPenalties } = require('../../../../../app/processing/siti-agri/handle-known-defects/cap-bps-penalties/recalculate-bps-penalties')
+
+describe('Identify if bps penalties need correcting', () => {
+  let paymentRequest
+  let invoiceLine
```

Comment:
Should the lets be defined outside the describe? - unless moving them outside interferes with the test cases below

Reply:
I can never remeber the structure of the test files and im scared of upsetting jest :fearful:
Have updated as suggested 😃